pkg/localcommit/localcommit.go

```diff
@@ -32,7 +34,7 @@ func Get(pjpath string) (strfmt.UUID, error) {
 	}
 
 	if !strfmt.IsUUID(proj.LegacyCommitID()) {
-		return "", locale.NewInputError("err_commit_id_invalid", "Commit ID in activestate.yaml is not UUID formatted: {{.V0}}.", proj.LegacyCommitID())
+		return "", &InvalidCommitID{locale.NewInputError("err_commit_id_invalid", "", proj.LegacyCommitID())}
```

Comment:
Note, the `err_commit_id_invalid` key is in `en-us.yaml`, so no need for this default text.